### ISSUE TITLE
Applications in VFX

### ISSUE THREAD

**Issue by emlcpfx** [qianqianwang68/omnimotion#10]
This research looks amazing.

Could this be used to export Nuke-like SmartVectors to 32-bit EXRs?

Could it be used for camera matchmove?

Very excited for how this could be applied to visual effects.

**Comment by hkaramian** [comment]
Anyone knows how we can convert it to nuke SmartVectors !? It would be very useful

**Comment by emlcpfx** [comment]
Check out this implementation of co-tracker:
https://www.linkedin.com/feed/update/urn:li:activity:7149755118165401600?updateEntityUrn=urn%3Ali%3Afs_feedUpdate%3A%28V2%2Curn%3Ali%3Aactivity%3A7149755118165401600%29

**Comment by hkaramian** [comment]
I have already seen this, but there is no information about implementation, and the code is not available yet. co-tracker output can't be used as nuke smart vector.

**Comment by emlcpfx** [comment]
Yup, I just thought it was cool. I'd love to get stmaps out of co-tracker or any of the other similar repos as well. Maybe one day.
